src/solver/tron.jl

```diff
@@ -229,10 +236,9 @@ function cauchy{T <: Real}(x::AbstractVector{T},
       end
       # TODO: Correctly assess why this fails
       if α < sqrt(nextfloat(zero(α)))
-        throw("α too small (qs = $qs, slope = $slope)")
-        #stalled = true
-        #status = "α too small"
-        #break
+        stalled = true
```

Comment:
This warrants a log message I think, at the warning or error level.

Reply:
I ended up returning the Cauchy status to the main function and issuing a `@critical` message there. Is it better to do it here?

Reply:
Oh I see. Sorry. That's all good. Thanks.